### PR TITLE
Styling Enhancements for Search Bar

### PR DESCRIPTION
**Pull request by avdaredevil**
### Things Enhanced in relation to #9

> - Input color and background will now adjust with parent's styles.
> - Styles adjusted to accommodate dynamic styles with:
>   - `--icon-styles`
>   - `--input-styles`
>   - `--disabled-input-styles`

### CODE REVIEW

File: paper-search-bar.html [hunk 1]
```diff
@@ -42,10 +43,13 @@
 				outline: 0;
 				padding: 16px 0;
 				cursor: text;
+				background: transparent;
+				color: inherit;
+                @apply(--input-styles);
```
Comment (ronnyroeller) [inline paper-search-bar.html]: Can you please use tabs instead of spaces to keep the code consistent?

File: paper-search-bar.html [hunk 2]
```diff
@@ -42,10 +43,13 @@
...
 			}
 
 			#input[disabled] {
-				background: var(--background-color, white);
+                @apply(--disabled-input-styles);
```
Comment (ronnyroeller) [inline paper-search-bar.html]: Can you please use tabs instead of spaces to keep the code consistent?

**Review by ronnyroeller — CHANGES_REQUESTED** [review]

**Comment by ankon** [comment]
FWIW: The travis failure was caused by https://github.com/Polymer/polymer-analyzer/issues/205, I've disabled polylint checking now.

File: paper-search-bar.html (hunk 2 above)
Comment (avdaredevil) [reply paper-search-bar.html]: Sure thing!

**Comment by avdaredevil** [comment]
Ok Tabs are fixed!

**Comment by avdaredevil** [comment]
@ronnyroeller @ankon, What's holding up the pull request?

**Comment by ronnyroeller** [comment]
Thanks for pinging, @avdaredevil . This got somehow lost.

**Comment by avdaredevil** [comment]
It happens! Thanks for the component!

**Pull request merged**